structuredefinitions/Extension-UKCore-NotificationHeaderVersion.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-NotificationHeaderVersion" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/Extension-UKCore-NotificationHeaderVersion" />
```

Comment:
This should be the HL7 extension, `http://hl7.org/fhir/5.0/StructureDefinition/Extension-<Resource>.<element>.<subelement>`